doc/user_guide/transformations.rst

```diff
@@ -211,7 +211,7 @@ can be found in the API-specific sections).
 
 ####
 
```

Comment:
I actually kind of would mind, since I am not exactly sure what the author wants to show. Well, I did fix it, but by adding significantly more lines (in order to show all the imports, which were moved to the top of the file). That example has become quite a bit more complex, and someone should have a look if we need a simpler example here. But the warning is gone :grin:

Reply:
Thanks Joerg. I've had a look and think it's OK for now - there's no avoiding the fact that doing OpenCL is quite complex at the moment. Possibly we should have a single, uber transformation that does everything but we're not there yet.